x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts.ts

```diff
@@ -61,7 +62,11 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       const fieldOptions = await find.allByCssSelector('#thresholdTimeField option');
       await fieldOptions[1].click();
       await nameInput.click();
-      await testSubjects.click('.slack-ActionTypeSelectOption');
+      await find.clickByCssSelector('[data-test-subj="intervalInput"]');
+      // await load action types to complete
+      await retry.try(async () => {
+        await testSubjects.click('.slack-ActionTypeSelectOption');
```

Comment:
nit: the `testSubjects.click(...)` function takes a second argument of `timeout`. I believe `retry.try` just has a bigger timeout than the default in `testSubjects.click(...)`.